src/mmw/apps/modeling/mapshed/tasks.py

```diff
@@ -156,6 +157,9 @@ def collect_data(geop_result, geojson):
 
     z.AvSlope = geop_result['avg_slope']
 
+    z.LS = (ls_factors(geop_result['lu_stream_pct'], z.StreamLength,
```

Comment:
This would be the correct indentation:

```python
    z.LS = ls_factors(geop_result['lu_stream_pct'], z.StreamLength,
                      z.Area, z.AvSlope)
```